burgos_bank.py

```diff
@@ -0,0 +1,17 @@
+class BurgosBank:
```

Comment:
O nome da classe deveria ser BurgosAgency
```suggestion
class BurgosAgency:
```